backend/battle/tests/test_view.py

```diff
@@ -74,3 +80,343 @@ def test_verify_if_not_returns_winner_missing_pokemons_teams(self):
+                    "attack": 25,
+                    "hp": 15,
+                    "name": 'pidgey',
+                    "img_url": 'https://raw.githubusercontent.com'
+                            '/PokeAPI/sprites/master/sprites/pokemon/25.png',
+                    "pokemon_id": 15,
+                }
+            elif pokemon_name == 'bulbasaur':
+                fake_json = {
+                    "defense": 30,
+                    "attack": 40,
+                    "hp": 20,
+                    "name": 'bulbasaur',
+                    "img_url": 'https://raw.githubusercontent.com'
+                            '/PokeAPI/sprites/master/sprites/pokemon/25.png',
+                    "pokemon_id": 10,
+                }
+            return fake_json
+        mock_get_pokemon.side_effect = side_effect_func
+
+        total_point_pokemons = 0
+
+        for pokemon in ['pikachu', 'pidgey', 'bulbasaur']:
+            pokemon_point = sum_point_from_api(pokemon)
+            mock_get_pokemon.assert_called_with(pokemon)
+            total_point_pokemons += pokemon_point
+
+        self.assertEqual(total_point_pokemons, 310)
+
+        is_valid_sum = validate_sum_pokemons(['pikachu', 'pidgey', 'bulbasaur'])
+        self.assertTrue(is_valid_sum)
+
+    @patch('battle.battles.battle.get_pokemon_from_api')
+    def test_if_function_valid_points_returns_corretly(self, mock_get_pokemon):
+        def side_effect_func(pokemon_name):
+            fake_json = 1
+            if pokemon_name == 'pikachu':
+                fake_json = {
+                    "defense": 40,
+                    "attack": 55,
+                    "hp": 35,
+                    "name": 'pikachu',
+                    "img_url": 'https://raw.githubusercontent.com'
+                            '/PokeAPI/sprites/master/sprites/pokemon/25.png',
+                    "pokemon_id": 25,
+                }
+            elif pokemon_name == 'pidgey':
+                fake_json = {
+                    "defense": 50,
+                    "attack": 25,
+                    "hp": 15,
+                    "name": 'pidgey',
+                    "img_url": 'https://raw.githubusercontent.com'
+                            '/PokeAPI/sprites/master/sprites/pokemon/25.png',
+                    "pokemon_id": 15,
+                }
+            elif pokemon_name == 'bulbasaur':
+                fake_json = {
+                    "defense": 30,
+                    "attack": 40,
+                    "hp": 20,
+                    "name": 'bulbasaur',
+                    "img_url": 'https://raw.githubusercontent.com'
+                            '/PokeAPI/sprites/master/sprites/pokemon/25.png',
+                    "pokemon_id": 10,
+                }
+            return fake_json
+        mock_get_pokemon.side_effect = side_effect_func
+
+        is_valid_sum = validate_sum_pokemons(['pikachu', 'pidgey', 'bulbasaur'])
+
+        self.assertTrue(is_valid_sum)
+
+
+class TeamViewTest(TestCaseUtils):
```

Comment:
I missed some tests here regarding the association of pokemon with the team. Is this is another PR?

Reply:
Yup! There's in another PR :)